### ISSUE TITLE
Ulisse 13 DCI Eco WiFi

### ISSUE THREAD

**Issue by Rudoxxxx** [nyffchanium/argoclima-integration#7]
## Configuration

```yaml
Add your logs here.
```

## Debug log

```text

Add your logs here.

```

**Comment by Rudoxxxx** [comment]
Only with this Argo Clima model working   Ulisse 13 DCI Eco WiFi?

**Comment by nyffchanium** [comment]
The Ulisse 13 DCI Eco WiFi is the only device I own, so I don't know for sure.

For the integration to work, the device needs to have a webinterface like the one below. See also #5.

![image](https://user-images.githubusercontent.com/55743116/233808067-c670c17a-c45b-4dba-9f60-63cd90bb9615.png)

**Comment by Rudoxxxx** [comment]
What do we need to change for another device? How U  get connections? We have AC air cond the model Argo Clima Greenstyle plus 18. Can We try to update the code?

**Comment by Rudoxxxx** [comment]
Nice Job!  Can we try and make the next step for this code for all Argo models?

**Comment by Rudoxxxx** [comment]
Yes, Greenstyle Plus uses the "AC Freedom" app. But the timer not working correctly ir U change timers every day
